bigquery_etl/feature_usage/templates/metadata.yaml

```diff
@@ -14,7 +14,7 @@ labels:
   application: firefox
```

Comment:
I wonder if it's worth updating the description with anything new here  (e.g. the links to the probe dictionary)? You may also want to add yourself to `owners`